shub/settings/config.py

```diff
@@ -96,11 +115,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.postgresql_psycopg2",
-        "NAME": "postgres",
-        "USER": "postgres",
-        "HOST": "db",
-        "PORT": "5432",
+        "ENGINE": os.environ.get("SREGISTRY_DATABASE_ENGINE")
```

Comment:
Consider leveraging the optional default arg of `.get()`:
```python
os.environ.get("SREGISTRY_DATABASE_ENGINE", "django.db.backends.postgresql_psycopg2")
```

Reply:
I've actually had times when it's set (but empty) so what I typically do to get around that is:

```python
os.environ.get('key') OR "default"
```